### PR TITLE
complete release flow and add reingest

### PR DESCRIPTION
**Pull request by doramatadora**
## Why?

This addresses issues [179](https://github.com/Financial-Times/runbook.md/issues/179),  [178](https://github.com/Financial-Times/runbook.md/issues/178),  [177](https://github.com/Financial-Times/runbook.md/issues/177), completing the new release flow and adding manual re-ingest capabilities.

## What has changed?

- fixed some snags on the release handling logic
- adds a re-ingest button on each runbook in the `runbook.md` Github app check run, on the following conditions:
1. the runbook has passed validation
2. the runbook has a system code assigned - otherwise, displays a notification about manual reingest being disabled with the intent to include a link to documentation explaining how to specify system code (another PR)

## Anything in particular you'd like to highlight to reviewers?

Clicking on the link to re-ingest (`Trigger ingest`) will open runbook.md `reingest` page in the browser (see `reingestHandler`), which triggers a Biz-Ops update if all conditions are met. Below is an example of when conditions are not met; the only difference for a valid runbook is a green notification if the update was successful: 

Screenshot 2019-11-22 at 14 28 16

## Scope and particulars of this PR (Please tick all that apply)

-   [ ] Tech hygiene (dependency updating & other tech debt)
-   [ ] Bug fix
-   [ ] Feature
-   [ ] Documentation / Runbook update
-   [ ] Breaking change
-   [ ] Minor change (e.g. fixing a typo, adding config)
-   [ ] Review not required

## Reviewer checklist

-   [ ] Check this box if you feel you fully understand this PR
-   [ ] Has the documentation been updated (if applicable)?

### CODE REVIEW

File: lambdas/ingester/src/statusPageHandler.js [hunk 1]
```diff
@@ -64,7 +64,10 @@ const statusPageHandler = async event => {
 			event: 'FAILED_FETCH_STORED_RESULT',
 			error,
 		});
-		// TODO error handling
+		return json(404, {
+			message:
+				'Failed to fetch stored validation result. Please make a fresh commit.',
```
Comment (wheresrhys) [inline lambdas/ingester/src/statusPageHandler.js]:
```suggestion
				'Failed to fetch stored validation result. This is probably our fault. Please make a fresh commit to force regeneration. If the problem persists, let us know in the #reliability-eng slack channel.',
```

File: lambdas/ingester/src/releaseHandler.js [hunk 2]
```diff
@@ -48,7 +48,10 @@ const fetchRunbooksByCommit = async (
 		// 1. make sure we have a repository; format owner/repositoryName
 		const repository = parseRepositoryName(gitRepositoryName, gitRefUrl);
 		// 2. make sure we have some runbooks saved in DynamoDb
-		const { runbookHashes } = await getStoredResult(repository, commit);
+		const { runbookHashes, checkRunUrl } = await getStoredResult(
```
Comment (wheresrhys) [inline lambdas/ingester/src/releaseHandler.js]: what's checkRunUrl

Comment (doramatadora) [reply lambdas/ingester/src/releaseHandler.js]: It's the url of this `Details...` page on github:
Screenshot 2019-11-22 at 14 43 51

**Comment by doramatadora** [comment]
@wheresrhys I'll make sure to document the hell out of this before next week, promised. The flow is now a lot less complex than it used to be. :bow:

**Pull request merged**